charts/trino/templates/autoscaler.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: autoscaling/v2
 kind: HorizontalPodAutoscaler
 metadata:
   name: {{ template "trino.worker" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
what happens if you do not specify namespace when installing helm? Will that resolve to `default`?

Reply:
I asked some clarifying questions in the issue: https://github.com/trinodb/charts/issues/114#issuecomment-1996286069

Reply:
Thank you for your reviews!

> what happens if you do not specify namespace when installing helm? Will that resolve to default?

Yes, it will resolve to `default` namespace.